### PR TITLE
[MIG]: Migrate crm_project_issue in new api.

### PR DESCRIPTION
**Pull request by atp-odoo**
Task: https://www.odoo.com/web#id=12140&view_type=form&model=project.task&menu_id=3942&action=327
pad: https://pad.odoo.com/p/r.FvY4KjvA7Amr2gKM

### CODE REVIEW

File: addons/crm_project_issue/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
 #
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2014-Today OpenERP SA (<http://www.openerp.com>).
+#    OdooERP, Open Source Management Solution
```
Comment (atp-odoo) [inline addons/crm_project_issue/__init__.py]: OdooERP ?

File: addons/crm_project_issue/project_issue.py [hunk 2]
```diff
@@ -55,5 +50,5 @@ def action_lead_to_project_issue(self, cr, uid, ids, context=None):
             'res_model': 'project.issue',
             'type': 'ir.actions.act_window',
             'res_id': issue_id,
-            'context': context
+            'context': self._context
```
Comment (atp-odoo) [inline addons/crm_project_issue/project_issue.py]: use self.env.context

File: addons/crm_project_issue/project_issue.py [hunk 3]
```diff
@@ -40,13 +35,13 @@ def action_lead_to_project_issue(self, cr, uid, ids, context=None):
...
+            issue_id = Issue.create(vals).id
             # move the mail thread
-            Lead.message_change_thread(cr, uid, lead.id, issue_id, "project.issue", context=context)
+            lead.message_change_thread(issue_id, "project.issue")
             # delete the lead
-            Lead.unlink(cr, uid, [lead.id], context=None)
+            lead.unlink()
         # return the action to go to the form view of the new Issue
-        view_id = self.pool.get('ir.ui.view').search(cr, uid, [('model', '=', 'project.issue'), ('name', '=', 'project_issue_form_view')])
+        view_id = self.env.ref('project_issue.project_issue_form_view').id
```
Comment (atp-odoo) [inline addons/crm_project_issue/project_issue.py]: good

File: addons/crm_project_issue/project_issue_view.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<openerp>
+<odoo>
 <data>
```
Comment (atp-odoo) [inline addons/crm_project_issue/project_issue_view.xml]: <data> ?

File: addons/crm_project_issue/project_issue.py [hunk 5]
```diff
@@ -1,35 +1,30 @@
...
+        Issue = self.env['project.issue']
 
-    def action_lead_to_project_issue(self, cr, uid, ids, context=None):
-        # get the wizards and models
-        wizards = self.browse(cr, uid, ids, context=context)
-        Lead = self.pool["crm.lead"]
-        Issue = self.pool["project.issue"]
-
-        for wizard in wizards:
+        for wizard in self:
```
Comment (hmo-odoo) [inline addons/crm_project_issue/project_issue.py]: if you use self.ensure_one(), no need "for loop"

**Comment by hmo-odoo** [comment]
rename project_issue.py with crm_lead2projectissue_wizard.py and move into models folder.
rename project_issue.xml with crm_lead2projectissue_wizard.xml and move into views folder.

**Comment by atp-odoo** [comment]
Make separate commit for move directory file
Change xml file name based on new guidlines - crm_lead2projectissue_wizard_views.xml.

**Pull request closed without merging**